src/fr/invinciblecomics/src/eu/kanade/tachiyomi/extension/fr/invinciblecomics/InvincibleComics.kt

```diff
@@ -114,12 +116,32 @@ class InvincibleComics : HttpSource() {
             ?: error("Failed to extract total pages from script")
 
         return (1..totalPages.toInt()).map { pageNumber ->
-            Page(pageNumber, imageUrl = "$imageBaseUrl${"%03d".format(pageNumber)}.png")
+            Page(pageNumber, "$imageBaseUrl${"%03d".format(pageNumber)}.png")
```

Comment:
Should try `.jpg` first.

Reply:
That's what the website does. I guess they just upload either of JPEG and PNG so only one would succeed. You have examples where both exist?

Reply:
I didn't found any examples so I rolled back